tools/release/backport_pr.sh

```diff
@@ -24,59 +24,104 @@ ensure_command () {
   fi
 }
 
+display_usage () {
+  cat << EOF >/dev/stderr
+USAGE: $0 PR_ID GITHUB_USER BACKPORT_BRANCHES REVIEWERS [-c PER_BACKPORT_COMMAND]
+   PR_ID: The ID of the PR to be backported.
+   GITHUB_USER: Your GitHub username.
+   BACKPORT_BRANCHES: A space-separated list of branches to which the
+     source PR will be backported.
+   REVIEWERS: A comma-separated list of users to add as both reviewer and assignee. >/dev/stderr
```

Comment:
There's an extra `>/dev/stderr` at the end here.

Reply:
🤦